feat: Add options for setting branch and changelog_file using action …

miigotu opened the pull request:
…env variables CHANGELOG_FILE and BRANCH

miigotu committed:
feat: Add options for setting branch and changelog_file using action env variables CHANGELOG_FILE and BRANCH

bjoluc commented:
Thanks @miigotu, this makes sense to be supported. In it's current form, however, it will (a) be a breaking change (the new variables are required) and (b) will not work since `$CHANGELOG_FILE` and `$BRANCH` are not substituted in the JSON file. I just released v2 which exports a JavaScript module instead of a JSON file, allowing us to (a) set defaults for the environment variables and (b) use the values of `CHANGELOG_FILE` and `BRANCH` in the exported config. I'll rebase this PR today if you don't mind.

bjoluc committed:
Fix implementation of the new variables

Also rename `branch` to `release_branch` and document both variables in the readme

bjoluc committed:
Fix changelog_file description in action.yml

miigotu commented:
Absolutely.
